transactor/src/main/scala/io/mediachain/transactor/RpcService.scala

```diff
@@ -30,13 +30,11 @@ object RpcService {
     logger.info(s"Connecting to cluster at $cluster...")
     client.connect(cluster)
 
-    implicit val ec = ExecutionContext.global
-    // todo: dynamic
-    val executor = Executors.newFixedThreadPool(4)
-    val datastoreConfig = DynamoDatastore.Config(
-      "Mediachain",
-      Some("http://localhost:8000")
-    )
+    val threads = Math.max(4, Runtime.getRuntime.availableProcessors)
+    implicit val ec = ExecutionContext.fromExecutor(Executors.newFixedThreadPool(threads))
```

Comment:
actually `TransactorService` takes both `executor: ExecutorService` and `implicit val executionContext: ExecutionContext`, which seems unnecessary. Can leave it be for now but something to clean up in the future.